docs/service_architecture.md

```diff
@@ -104,53 +103,57 @@ sever, and signer:
- 	or delete metadata from the database, thus preventing clients from being
+    or delete metadata from the database, thus preventing clients from being
     able to download or upload metadata.
 
 - **Malicious Content** - An attacker can create, store, and serve arbitrary
-    metadata content for one or more repositories. However, they do not have
-    access to the original root, target, or (maybe) snapshots keys for
-    existing repositories.
+    metadata content for one or more trusted collections. However, they do not have
+    access to any client-side keys, such as root, targets, and potentially the
+    snapshot keys for the existing trusted collections.
 
-    Only clients who have never seen, and do not have any form of pinned trust
-    for, the compromised repositories can be tricked into downloading and
-    trusting the malicious content for these repositories.
+    Only clients who have never seen the trusted collections, and who do not have any
+    form of pinned trust, can be tricked into downloading and
+    trusting the malicious content for these trusted collections.
 
-    Clients who have pinned trust for the compromised repositories, either
-    due to configuration or due to TOFU (trust on first use), will immediately
+    Clients that have previously interacted with any trusted collection, or that have
+    their trust pinned to a specific certificate for the collections will immediately
     detect that the content is malicious and would not trust any root, targets,
-    or (maybe) snapshot metadata for these repositories.
+    or (maybe) snapshot metadata for these collections.
 
- - **Rollback, Mix and Match** - The attacker can request that
-    the Notary signer sign whatever timestamp (and maybe snapshot) metadata
-    they want. They can create valid timestamp and snapshot metadata that
-    certifies that the latest version of the repository contains
-    strictly older, or even a mix of older and newer, data.
+- **Rollback, Freeze, Mix and Match** - The attacker can request that
+    the Notary signer sign any arbitrary timestamp (and maybe snapshot) metadata
+    they want. Attackers can lauch a freeze attack, and, depending on whether
+    the snapshot key is available, a mix-and-match attack up to the expiration
+    of the targets file.
 
     Clients both with and without pinned trust would be vulnerable to these
-    attacks.
+    attacks, so long as the attacker ensures that the version number of their
+    malicious metadata is higher than the version number of the most recent
+    good metadata that any client may have.
 
- Note that the timestamp and snapshot keys are never compromised; once the
- Server compromise is mitigated, an attacker will not be able to generate
- valid timestamp or snapshot metadata and serve them on a malicious mirror, for
- example.
+ Note that the timestamp and snapshot keys cannot be compromised in a server-only
```

Comment:
Ah ok, cool thanks.  Did we want to move the threat model stuff into another doc page as you suggested in the previous PR?